templates/docs/examples/utilities/truncate.html

```diff
@@ -35,4 +35,14 @@
     </tbody>
   </table>
   <p class="u-truncate">This sentence will truncate and reveal an ellipsis once it exceeds the paragraph's max-width.</p>
+
+  <div class="row">
+    <div class="col-3">
+      <label class="p-checkbox u-truncate">
+        <input type="checkbox" class="p-checkbox__input">
+        <span class="p-checkbox__label">This label will truncate at the end of the col</span>
+      </label>
+    </div>
+  </div>
```

Comment:
We can also do this, it looks a little neater on the page:
```suggestion
  <label class="p-checkbox u-truncate" style="width: 20rem;">
    <input type="checkbox" class="p-checkbox__input">
    <span class="p-checkbox__label">This label will truncate at the end of the col</span>
  </label>
```